mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/massindexing/impl/PojoDefaultMassIndexer.java

```diff
@@ -122,8 +124,11 @@ public PojoDefaultMassIndexer monitor(MassIndexingMonitor monitor) {
 	public CompletableFuture<?> start() {
 		PojoMassIndexingBatchCoordinator coordinator = createCoordinator();
 		ExecutorService executor = mappingContext.threadPoolProvider()
-				.newFixedThreadPool( 1,
-						PojoMassIndexingBatchIndexingWorkspace.THREAD_NAME_PREFIX + "Coordinator" );
+				.newFixedThreadPool(
+						1,
+						PojoMassIndexingBatchIndexingWorkspace.THREAD_NAME_PREFIX + "Coordinator"
+						// todo: double check if we need any context around the coordinator. seems like no, but .. :)
```

Comment:
I don't think so. IMO the context is only useful for threads where we list identifiers or load entity data.